phoenix-core/src/it/java/org/apache/phoenix/monitoring/PhoenixTableLevelMetricsIT.java

```diff
@@ -1276,14 +1281,24 @@ public void testMetricsWithIndexUsage() throws Exception {
                 assertTrue(metricExists);
                 metricExists = false;
                 //assert BaseTable is not being queried
-                for (PhoenixTableMetric metric : getPhoenixTableClientMetrics().get(dataTable)) {
-                    if (metric.getMetricType().equals(SELECT_SQL_COUNTER)) {
-                        metricExists = true;
-                        assertMetricValue(metric, SELECT_SQL_COUNTER, 0, CompareOp.EQ);
-                        break;
+                //if client is validating last_ddl_timestamps with ucf=never,
+                //there will be no metrics for base table (like getTable RPC times/counts).
+                boolean lastDDLTimestampValidationEnabled =
+                        conn.unwrap(PhoenixConnection.class).getQueryServices().getProps().getBoolean(
```

Comment:
Do you want to use [ValidateLastDDLTimestampUtil#getValidateLastDdlTimestampEnabled](https://github.com/apache/phoenix/blob/PHOENIX-6883-feature/phoenix-core-client/src/main/java/org/apache/phoenix/util/ValidateLastDDLTimestampUtil.java#L73C27-L73C61) method instead?